inc/target.class.php

```diff
@@ -137,14 +137,20 @@ public function prepareInputForAdd($input)
 
                if (!isset($input['_skip_create_actors'])
                    || !$input['_skip_create_actors']) {
-                  $query = "INSERT INTO glpi_plugin_formcreator_targettickets_actors
-                            (`plugin_formcreator_targettickets_id`, `actor_role`, `actor_type`, `use_notification`)
-                            VALUES (
-                               $id_targetticket, 'requester', 'creator', 1
-                            ), (
-                               $id_targetticket, 'observer', 'validator', 1
-                            );";
-                  $DB->query($query);
+                  $targetTicket_actor = new PluginFormcreatorTargetTicket_Actor();
+                  $targetTicket_actor->add(array(
+                        'plugin_formcreator_targettickets_id'  => $id_targetticket,
+                        'actor_role'                           => 'requester',
+                        'actor_type'                           => 'creator',
+                        'use_notification'                     => '1'
+                  ));
+                  $targetTicket_actor = new PluginFormcreatorTargetTicket_Actor();
```

Comment:
I do not think this second one is needed, you should you call again add() method.